### PR TITLE
A note about leveraging JJ's ignite karaoke app

### PR DESCRIPTION
**Pull request by jjasghar**
Instead of brining it into devopsdays github, here's a link to the app to make organizers life easier.

### CODE REVIEW

File: content/page/organizing.md [hunk 1]
```diff
@@ -712,6 +712,8 @@ Running Ignites can be a challenge: people tend to submit them last minute, or t
 
 [Ignite format page](/ignite-talks-format/)
 
+If you are looking for a way to fill in for more ignites (pad the schedule) or have some fun with volunteers "Ignite Karaoke" is a great way to engage with your audiance. The idea is just one minute per volunteer, 4 slides 15 seconds each, with a title and some "funny" pictures. It works extremely well for some laughs and engagement, and people can enjoy it greatly. JJ Asghar has created a tool to help create these slides if you want to experiment with this new way expose more people to the ignite format in a fun manner. [GitHub repo](https://github.com/jjasghar/ignite_karaoke)
```
Comment (adrianmoisey) [inline content/page/organizing.md:715]:
```suggestion
If you are looking for a way to fill in for more ignites (pad the schedule) or have some fun with volunteers "Ignite Karaoke" is a great way to engage with your audience. The idea is just one minute per volunteer, 4 slides 15 seconds each, with a title and some "funny" pictures. It works extremely well for some laughs and engagement, and people can enjoy it greatly. JJ Asghar has created a tool to help create these slides if you want to experiment with this new way expose more people to the ignite format in a fun manner. [GitHub repo](https://github.com/jjasghar/ignite_karaoke)
```

**Review by jyee — COMMENTED** [review]
LGTM.
Normally I hate PRs that modify more than what's being added/removed, but I appreciate that this is also removing trailing whitespace. :)

**Review by jyee — APPROVED** [review]

**Comment by jjasghar** [comment]
squashed :)

File: content/page/organizing.md [hunk 2]
```diff
@@ -712,6 +712,8 @@ Running Ignites can be a challenge: people tend to submit them last minute, or t
 
 [Ignite format page](/ignite-talks-format/)
 
+If you are looking for a way to fill in for more ignites (pad the schedule) or have some fun with volunteers "Ignite Karaoke" is a great way to engage with your audience. The idea is just one minute per volunteer, 4 slides 15 seconds each, with a title and some "funny" pictures. It works extremely well for some laughs and engagement, and people can enjoy it greatly. JJ Asghar has created a tool to help create these slides if you want to experiment with this new way expose more people to the ignite format in a fun manner. [GitHub repo](https://github.com/jjasghar/ignite_karaoke)
```
Comment (phrawzty) [inline content/page/organizing.md:715]:
```suggestion
If you're looking for a way to fill in for more ignites (i.e. pad the schedule), or just to have some fun, "Ignite Karaoke" is a great way to engage with your audience. The idea is straightforward: one minute per participant, four slides of 15 seconds each, where each slide has a title and a "funny" picture. It's great for some laughs and engagement—people really enjoy it! JJ Asghar maintains a tool to help create these slides if you want to experiment with this fun ignite format: [GitHub repo](https://github.com/jjasghar/ignite_karaoke)
```

**Review by phrawzty — CHANGES_REQUESTED** [review]
Agree with content, suggest mild rewording

**Comment by jjasghar** [comment]
I'll have to squash it again.

**Comment by adrianmoisey** [comment]
> I'll have to squash it again.

Don't worry about squashing, we normally squash and merge when merging into main.

**Review by phrawzty — APPROVED** [review]

**Comment by phrawzty** [comment]
Thanks @jjasghar ! <3

**Pull request merged**